Formula/aerc.rb

```diff
@@ -17,6 +18,7 @@ class Aerc < Formula
 
   def install
     system "make", "PREFIX=#{prefix}"
+    inreplace "aerc.conf", pkgshare.to_s, "#{HOMEBREW_PREFIX}/share/aerc"
```

Comment:
This looks like a bug in the install script. What exactly is the issue?

Reply:
it writes a default aerc.conf file and puts the full share path in it. like `/home/linuxbrew/.linuxbrew/Cellar/aerc/0.5.1_1/share`. and if someone uses this as a own config file base the path would be invalid after every update.

Reply:
and i want it to be `/home/linuxbrew/.linuxbrew/share/aerc`.

Reply:
Shouldn't that file be writen to `#{HOMEBREW_PREFIX}/etc`? Because then you should replace it there to avoid it being overwritten.

Reply:
you are right. but i just use their `make install`. and it looks like this:

```
install: all
	mkdir -m755 -p $(BINDIR) $(MANDIR)/man1 $(MANDIR)/man5 $(MANDIR)/man7 \
		$(SHAREDIR) $(SHAREDIR)/filters $(SHAREDIR)/templates
	install -m755 aerc $(BINDIR)/aerc
	install -m644 aerc.1 $(MANDIR)/man1/aerc.1
	install -m644 aerc-search.1 $(MANDIR)/man1/aerc-search.1
	install -m644 aerc-config.5 $(MANDIR)/man5/aerc-config.5
	install -m644 aerc-imap.5 $(MANDIR)/man5/aerc-imap.5
	install -m644 aerc-maildir.5 $(MANDIR)/man5/aerc-maildir.5
	install -m644 aerc-sendmail.5 $(MANDIR)/man5/aerc-sendmail.5
	install -m644 aerc-notmuch.5 $(MANDIR)/man5/aerc-notmuch.5
	install -m644 aerc-smtp.5 $(MANDIR)/man5/aerc-smtp.5
	install -m644 aerc-tutorial.7 $(MANDIR)/man7/aerc-tutorial.7
	install -m644 aerc-templates.7 $(MANDIR)/man7/aerc-templates.7
	install -m644 config/accounts.conf $(SHAREDIR)/accounts.conf
	install -m644 aerc.conf $(SHAREDIR)/aerc.conf
	install -m644 config/binds.conf $(SHAREDIR)/binds.conf
	install -m755 filters/hldiff $(SHAREDIR)/filters/hldiff
	install -m755 filters/html $(SHAREDIR)/filters/html
	install -m755 filters/plaintext $(SHAREDIR)/filters/plaintext
	install -m644 templates/quoted_reply $(SHAREDIR)/templates/quoted_reply
	install -m644 templates/forward_as_body $(SHAREDIR)/templates/forward_as_body
```
they want the `aerc.conf` in SHAREDIR. and in the `aerc.conf` they link to the filter files like `filters/hldiff`. what do you think?

Reply:
Sounds like it needs a PR upstream to allow it to install in default UNIX locations like `/etc`.

Reply:
now i get it... its a default config used to create a new config. when you start aerc without a local config, it starts a wizard asking for credentials and stuff... and it uses the config in the sharedir to create the local one. its more a template for a new config. so i think its valid or?

Reply:
I'd still install it to etc, but this way it doesn't sound quite as bad. I think a pull request to make this configurable upstream would still be a good idea though. I don't think homebrew would be the only package manager that's confused here.

Reply:
you are right... right now alpine for example handle this like we do with homebrew. so i would be happy about a merge because im using this :)

Reply:
and thanks for your review and suggestions... they help a beginner, like me, alot.